src/routes/index.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { createAgent, logAgent } from '../controlllers/AgentSignup';
+import { validateUserInput } from '../middleware';
+
+const indexRouter = express.Router();
```

Comment:
Let's give this file a better name, it can't be `index.js` anymore, it contains the route for our `Authentication` route, let's name it accordingly.